rskj-core/src/main/java/co/rsk/config/BridgeTestNetConstants.java

```diff
@@ -86,7 +86,7 @@
         minimumLockTxValue = Coin.valueOf(1000000);
         minimumReleaseTxValue = Coin.valueOf(500000);
 
-        // Keys generated with GenNodeKey using generators 'auth-a' through 'auth-e'
+        // Passphrases are kept private
```

Comment:
oh, yeas, who would look at the history? we should change the generated keys if we want this

Reply:
`"auth-a"` was never the real testnet key, but the regtest/devnet key. The comment here was outdated and removed because it was wrong and misleading.

We'd never commit private keys to a public repo :).

Reply:
Ah, ok.